frontend/src/Editor/Components/Table/load-properties-and-styles.js

```diff
@@ -10,6 +10,7 @@ export default function loadPropertiesAndStyles(properties, styles, darkMode, co
 
   const totalRecords = properties.totalRecords ?? '';
   const enabledSort = properties?.enabledSort ?? true;
+  const toggleShowColumns = properties?.toggleShowColumns ?? false;
```

Comment:
```suggestion
  const hideColumnSelectorButton = properties?.hideColumnSelectorButton ?? false;
```
Let's rename likewise everywhere else also